docs/docs/icicle/build_from_source.md

```diff
@@ -175,6 +175,4 @@ Make sure to install icicle libs when installing a library/application that depe
 
 ### Go: Build, Test, and Install (TODO)
 
-## Install cuda backend
-
-[Install CUDA Backend (and License)](./install_cuda_backend.md#installation)
\ No newline at end of file
+## [Install CUDA backend & License](./install_cuda_backend.md#installation)
```

Comment:
A bit strange to have a Header for a section that is a link and nothing in the section

Reply:
fixed